db/migrate/20191014155007_add_personal_details_fields_to_application_forms.rb

```diff
@@ -0,0 +1,10 @@
+class AddPersonalDetailsFieldsToApplicationForms < ActiveRecord::Migration[6.0]
+  def change
+    add_column :application_forms, :first_nationality, :string
+    add_column :application_forms, :second_nationality, :string
+    add_column :application_forms, :english_main_language, :string
```

Comment:
is it `main language` or should it be `native language`?

Reply:
So far we've named it `main_language` because that's what it's referred to in the form:

<img width="703" alt="Screenshot 2019-10-15 at 12 28 34" src="https://user-images.githubusercontent.com/1650875/66827778-563c0000-ef47-11e9-9bf6-cd48eb811645.png">